microsoft/testsuites/network/netinterface.py

```diff
@@ -45,7 +45,10 @@ class NetInterface(TestSuite):
 
         """,
         priority=1,
-        requirement=simple_requirement(network_interface=Synthetic()),
+        requirement=simple_requirement(
+            supported_features=[NetworkInterface],
```

Comment:
The network and storage are special features, they should be supported by platforms always. Please create them in platforms.

Reply:
Implementing NetworkInterface and Disk are on the TODO list. But they are too large for this change.